web/context.go

```diff
@@ -221,6 +221,10 @@ func (c *Context) SetInvalidParam(parameter string) {
 	c.Err = NewInvalidParamError(parameter)
 }
 
+func (c *Context) SetInvalidJSONParam(parameter string, err error) {
```

Comment:
This is not specific to json errors actually right? Maybe we can use a more generic name. 0/5

Reply:
I like the idea, how would you name the method?